invenio_cli/commands/containers.py

```diff
@@ -230,14 +233,23 @@ def translations(self, project_shortname):
         cmd = cmd[0].cmd  # extract compilation command
         cmd = " ".join(cmd)  # convert to string
 
+        print(cmd)
```

Comment:
seems correct, `pipenv run pybabel compile --directory=${INVENIO_INSTANCE_PATH}/translations`

Reply:
The translations files are there ([COPY command](https://github.com/inveniosoftware/cookiecutter-invenio-rdm/blob/master/%7B%7Bcookiecutter.project_shortname%7D%7D/Dockerfile#L21)). Do you have the actual error that is throwing? I have the suspicion that is because the [`skippable`](https://github.com/inveniosoftware/invenio-cli/blob/master/invenio_cli/commands/translations.py#L118) might not be working on containerized.